modules/openy_gc_auth/src/GCUserAuthorizer.php

```diff
@@ -72,11 +73,22 @@ public function authorizeUser($name, $email, array $extra_data = []) {
         $account->save();
       }
     }
+    // List of roles to redirect user login page.
+    $userRolesArray = [
+      'administrator',
+      'virtual_ymca_editor',
+    ];
+    // Redirecting user login page.
+    foreach ($userRolesArray as $role) {
+      if ($account->hasRole($role)) {
+        $response = new RedirectResponse('/user/login', 301);
+        $response->send();
```

Comment:
It is better to add Drupal message which is saying that you have to login as real user, sin you are an administrator.

Reply:
@anpolimus  I have added Drupal message, Can you please review.